livekit/src/room/participant/local_participant.rs

```diff
@@ -51,6 +67,9 @@ struct LocalEvents {
 struct LocalInfo {
     events: LocalEvents,
     encryption_type: EncryptionType,
+    pending_acks: Mutex<HashMap<String, oneshot::Sender<()>>>,
+    pending_responses: Mutex<HashMap<String, oneshot::Sender<Result<String, RpcError>>>>,
+    rpc_handlers: Mutex<HashMap<String, RpcHandler>>,
```

Comment:
Can you reduce the amount of Mutex here?

Reply:
yeah I moved these into a single object called RpcState and put a mutex on that